lib/nexpose/silo_profile.rb

```diff
@@ -107,7 +107,7 @@ def save(connection)
       begin
         update(connection)
       rescue APIError => error
-        raise error unless (error.message =~ /A silo profile .* does not exist./)
+        raise error unless (error.message =~ /The silo profile .* does not exist/)
```

Comment:
So #160's root cause is that the message for the error changed ever so slightly, and your fix is to ... change it here ever so slightly.  Why not:

```
raise error unless (error.message =~ /\S+ silo profile .* does not exist/
```

or something similarly more flexible?


(Also, OT, but Ideally there'd be a dedicated exception class for this so you could check the type rather than the message, but this is not an uncommon pattern, especially since the Nexpose gem is somewhat at the mercy of what Nexpose itself responds with)